.github/workflows/release.yaml

```diff
@@ -0,0 +1,153 @@
+name: Release Build
```

Comment:
The release pipeline is opinionated, I prefer to only support the cargo installed binary.